pkg/middlewares/auth/forward.go

```diff
@@ -132,6 +135,19 @@ func (fa *forwardAuth) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if fa.forwardBody {
```

Comment:
The forwarded request is Get, which does not support body.
If forwardBody is true, should we respect the incoming request method as well? or make that an option as well

Reply:
Thank you for reviewing and comment.
I updated the PR to respect method if forwardBody = true.